### DISCUSSION TITLE
How to use for Fedora Linux

### DISCUSSION THREAD

**Discussion by ArikRahman** [ValeLang/Vale#560] [Q&A]
Howdy! I was just wondering how you would compile this from source for use with Fedora?

**Comment by ArikRahman** [comment]
(I'm assuming I just download the Ubuntu release and use ./valec build, no?)

**Comment by ron-wolf** [comment]
I'll assume you've read [the guide](https://github.com/ValeLang/Vale/blob/master/build-compiler.md#ubuntu) on how to do this for Ubuntu already.

Being a Fedora user, do you already have some experience adapting installation instructions to your distro? For instance, are you acquainted with the helpful tools Fedora provides to help find equivalent packages, such as `dnf provides <program>`? If so, where specifically are you getting suck?
